src/main/java/games/strategy/engine/framework/startup/ui/editors/EmailSenderEditor.java

```diff
@@ -28,19 +28,19 @@
  */
 public class EmailSenderEditor extends EditorPanel {
   private static final long serialVersionUID = -4647781117491269926L;
-  private final GenericEmailSender m_bean;
-  private final JTextField m_subject = new JTextField();
-  private final JTextField m_toAddress = new JTextField();
-  private final JTextField m_host = new JTextField();
-  private final JTextField m_port = new JTextField();
-  private final JTextField m_login = new JTextField();
-  private final JCheckBox m_useTLS = new JCheckBox("Use TLS encryption");
-  private final JTextField m_password = new JPasswordField();
-  private final JLabel m_toLabel = new JLabel("To:");
-  private final JLabel m_hostLabel = new JLabel("Host:");
-  private final JLabel m_portLabel = new JLabel("Port:");
-  private final JButton m_testEmail = new JButton("Test Email");
-  private final JCheckBox m_alsoPostAfterCombatMove = new JCheckBox("Also Post After Combat Move");
+  private final GenericEmailSender genericEmailSender;
+  private final JTextField subject = new JTextField();
+  private final JTextField toAddress = new JTextField();
+  private final JTextField host = new JTextField();
+  private final JTextField port = new JTextField();
+  private final JTextField login = new JTextField();
+  private final JCheckBox useTLS = new JCheckBox("Use TLS encryption");
```

Comment:
If you feel like it, you could rename to `useTls` to also get rid of the abbreviation violation.